ament_tools/verbs/build/cli.py

```diff
@@ -225,6 +225,7 @@ def iterate_packages(opts, packages, per_package_callback):
     opts.skip_packages = opts.skip_packages or []
     install_space_base = opts.install_space
     package_dict = dict([(path, package) for path, package, _ in packages])
+    workspace_package_names = sorted([pkg.name for pkg in package_dict.values()])
```

Comment:
`sorted` isn't necessary since the result isn't used for a loop. For performance reasons I would suggest to remove it.